windows_installer_docker_build/Dockerfile

```diff
@@ -2,10 +2,6 @@ FROM ubuntu:xenial
 
 RUN apt-get -y update 
 
-# Install tff-mscorefonts so it won't prompt to accept the license later
-RUN echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | debconf-set-selections
-RUN apt-get install -y ttf-mscorefonts-installer
```

Comment:
These are license prohibited anyways

Reply:
`RUN echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true` this part will accept a EULA on behalf of a user, essentially - I would think - putting the blame of license violations on LE.

Reply:
Sorry this was for testing the buildkite-agent virtual image I'm working on currently. I've already removed them :)